components/Cards/README.md

```diff
@@ -14,28 +14,79 @@ api_doc_root: true
   <img src="docs/assets/cards.png" alt="Cards" width="375">
 </div>
 
-Cards provides an card control and a card collection view cell that adhere to Material Design
-layout and styling.
+Cards are contained surfaces that serve as an entry point to more detailed information, with flexible visual parameters.
+
+They can be used as a standalone UIControl, or as a UICollectionViewCell in a UICollectionView.
+Cards are meant to be interactive, and shouldn't be used solely for displaying purposes.
+
+Cards adhere to Material Design layout and styling.
 
 ## Design & API Documentation
 
 <ul class="icon-list">
-  <li class="icon-list-item icon-list-item--spec"><a href="https://material.io/guidelines/components/cards.html">Material Design guidelines: Cards</a></li>
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/cards.htm">Material Design guidelines: Cards</a>
+  </li>
 </ul>
 
+## Installation
+
+### Requirements
+
+- Xcode 8.0 or higher.
+- iOS SDK version 8.0 or higher.
+- To use the baked UICollectionView or UICollectionViewController reordering, then you must use iOS SDK version 9.0 or higher.
+
+### Installation with CocoaPods
+
+To add this component to your Xcode project using CocoaPods, add the following to your `Podfile`:
+
+```
+pod 'MaterialComponents/Cards'
+```
+
+Then run the following command:
+
+``` bash
+pod install
+```
+
+- - -
+
+### Overview
+
+Cards can be used either as a standalone `UIControl`, or as a `UICollectionViewCell`.
```

Comment:
Consider:
Cards provides two different versions, MDCCard inheriting from UIControl and MDCCollectionViewCell inheriting from UICollectionCell.